File: 1-Iniciante/1037.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main(void)
{
    double num;

    cin >> num;

    if (num >= 0 && num <= 25.0)
        cout << "Intervalo [0,25]" << endl;
    else if (num > 25.0 && num <= 50.0)
        cout << "Intervalo (25,50]" << endl;
    else if (num > 50.0 && num <= 75.0)
        cout << "Intervalo (50,75]" << endl;
    else if (num > 75.0 && num <= 100.0)
        cout << "Intervalo (75,100]" << endl;
    else
        cout << "Fora de intervalo" << endl;
    return 0;
}

File: 1-Iniciante/1037.py
num = float(input())

if (0 <= num <= 100):
    if (num <= 25.0): print('Intervalo [0,25]')
    elif (num <= 50.0): print('Intervalo (25,50]')
    elif (num <= 75.0): print('Intervalo (50,75]')
    else: print('Intervalo (75,100]')
else: print('Fora de intervalo')
